RosettaCore/src/main/java/com/hubspot/rosetta/RosettaMapper.java

```diff
@@ -58,14 +58,14 @@ public T mapRow(ResultSet rs) throws SQLException {
       final Object value;
       // calling getObject on a BLOB/CLOB produces weird results
       switch (metadata.getColumnType(i)) {
-        case Types.BLOB:
-          value = rs.getBytes(i);
-          break;
-        case Types.CLOB:
-          value = rs.getString(i);
-          break;
-        default:
-          value = rs.getObject(i);
+      case Types.BLOB:
```

Comment:
This seems weird, the body of the switch statement is inside braces and should increase the level of indentation